mri/operators/fourier/non_cartesian.py

```diff
@@ -13,26 +13,28 @@
 
 # System import
 import warnings
-import numpy as np
```

Comment:
Since we changed the distribution is it really necessary to change the import order?

Reply:
Hmm, I can change it back, but it would help to keep as a safeguard in my opinion